oaknational/Oak-Web-Application
# Feat/lesq 1017/hubspot form submit [LESQ-1017]

## Description

Music year: {owa_music_year}

- set up the onboarding form env vars
- create helper fn and types for generating onboarding hubspot form
- submit the form on completion of onboarding

## How to test

1. Go to https://deploy-preview-2705--oak-web-application.netlify.thenational.academy/onboarding
2. Click on yes / no
3. Complete the form and click continue
4. You should see your form is submitted to hubspot [here](https://app-eu1.hubspot.com/submissions/24947711/form/50399960-674f-4e23-ad66-367fbdbcdbfb/submissions/) containing the correct values

## Checklist

- [ ] The school picker form should submit correctly
- [ ] The role selection form should submit correctly
